vacationtime-solution/SampleData.csv

```diff
@@ -0,0 +1,5 @@
+sholmes@email.com, Sherlock Holmes, manager@gmail.com, June 20, June 25, Vacation with family
```

Comment:
This looks like the wrong sample data too.

Reply:
Fixed - correct sample data should be up

Reply:
I still see the wrong data here.